### DISCUSSION TITLE
Configuration: use strict, but disable some of rules

### DISCUSSION THREAD

**Discussion by tamuhey** [microsoft/pyright#1436] [Q&A] [answered]
I want pyright to check my code with `strict` mode, but want to disable some of rules. (e.g. reportMissingTypeStubs).
How to configure? Is only way to do that list all rules in `pyrightconfig.json`?

**Comment by erictraut** [comment, marked as answer]
You can specify the severity of individual diagnostic checks within the pyrightconfig.json. In strict mode, overrides will generally never make the rule less strict than the default used in strict mode. The one exception is `reportMissingModuleSource`. For that diagnostic rule, you can override the strict-mode setting.

For more details, refer to [this documentation](https://github.com/microsoft/pyright/blob/master/docs/configuration.md#diagnostic-rule-defaults).

Also, refer to [this part](https://github.com/microsoft/pyright/blob/39cb4cffb5a901902d1263cd34785e0fa2e93bc1/packages/pyright-internal/src/common/configOptions.ts#L315) of the pyright source code.
